config/template.yml

```diff
@@ -68,6 +68,16 @@ request:
     - <string>
     - ...
 
+  # The number of allowed requests per user per day.
+  # If the number of requests created by a single user exceeds this amount, new requests will not be forwarded to the internal channels.
+  # The length of this array MUST be the same as the length of `channels`.
+  # Setting to a negative number will remove the request limit
+  # Optional; empty by default.
```

Comment:
Out of scope for this PR, but I don't like this invariant very much. We could definitely structure the request config a lot nicer.

I think I'll work on that since I'll be reworking some stuff regarding request channels anyway.